implementation/pom.xml

```diff
@@ -69,6 +69,50 @@
           <forkMode>once</forkMode>
         </configuration>
       </plugin>
+      <plugin>
```

Comment:
Should something like this be moved into `smallrye-parent` so all projects could use it?

Reply:
I'd be okay with that I guess.

Reply:
But that means there'd have to be a release and so forth.

Reply:
True, but pulling the trigger is pretty easy.

Other than needing a release of parent, any reason not to?

Reply:
No, sounds fine.  Is that something you can do?

Reply:
Sure